assets/app/scripts/services/applicationGenerator.js

```diff
@@ -205,16 +205,7 @@ angular.module("openshiftConsole")
         });
       }
 
-      // User can input a URL that contains a ref
-      var uri = new URI(input.buildConfig.sourceUrl);
-      var sourceRef = uri.fragment();
-      if (!sourceRef || sourceRef.length === 0) {
```

Comment:
Not needed, I just forgot "" is falsey, I can't ever keep that one straight
in my head.

On Wed, Sep 23, 2015 at 11:02 AM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/scripts/services/applicationGenerator.js
> <https://github.com/openshift/origin/pull/4684#discussion_r40213956>:
>
> > @@ -205,16 +205,7 @@ angular.module("openshiftConsole")
> >          });
> >        }
> >
> > -      // User can input a URL that contains a ref
> > -      var uri = new URI(input.buildConfig.sourceUrl);
> > -      var sourceRef = uri.fragment();
> > -      if (!sourceRef || sourceRef.length === 0) {
>
> @jwforres <https://github.com/jwforres> Any reason the sourceRef.length
> check was needed here? Does fragment() not return a string?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4684/files#r40213956>.
>